students/14th/team_6/hyun/project_westagram/user/views.py

```diff
@@ -0,0 +1,76 @@
+from django.views import View
+from .models import User
+from django.http import JsonResponse
+import json
+
+class SignUpView(View):
+    def post(self,request):
+        signup_data = json.loads(request.body)
+        try:
+
+            if User.objects.all().filter(
+                user_name = signup_data["name"]).exists():
+                return JsonResponse(
+                { 'message: ' : 'ID already exists'}, status = 400)
+            elif User.objects.all().filter(
+                user_email = signup_data["email"]).exists():
+                return JsonResponse(
+                { 'message: ' : 'Email already exists'}, status = 400)
+            elif User.objects.all().filter(user_phone_number =
+                    signup_data["phone_number"]).exists():
+                return JsonResponse(
+                { 'message : ' :'phone_number already exists'} , status = 400)
+
+            if User.objects.all().filter(
+                user_email    = signup_data["email"],
+                user_password = signup_data["password"]
+                ).exists():
+                return JsonResponse ( {"message: " : "KEY_ERROR"}, status = 400)
+            elif  '@' not in signup_data["email"]  or '.' not in signup_data["email"] :
+                return JsonResponse(
+                    { "message: " : "Invalid email" }, status = 400 )
+            elif len(signup_data["password"]) < 8 :
+                return JsonResponse(
+                    { "message: " : "password validation" } , status = 400 )
+
+            User.objects.create(
+                user_name         = signup_data["name"],
+                user_email        = signup_data["email"],
+                user_phone_number = signup_data["phone_number"],
+                user_password     = signup_data["password"]
+            )
+
+            return JsonResponse ( { 'message: ' : '회원가입 성공!'} , status = 200 )
+
+        except KeyError :
+            return JsonResponse( {"message: " : "ERROR"} , status = 400 )
```

Comment:
```suggestion
        except KeyError :
            return JsonResponse( {"message: " : "KEY_ERROR"} , status = 400 )
```
발생하는 에러의 종류가 어떤 것인지 알려주는 것이 좋습니다